### DISCUSSION TITLE
Would this adapter work?

### DISCUSSION THREAD

**Discussion by tinkerer9** [aqualinkd/AqualinkD#308] [Announcements]
Hello World!

I was wondering if [this adapter](https://www.amazon.com/gp/product/B0BNZBNPWL/ref=ox_sc_act_title_2?smid=A3B0XDFTVR980O&th=1) would work. It says it has the genuine FT232RL chip, but I wasn't sure if it would work.

Thanks a lot!

**Comment by sfeakes** [comment]
Yes, I use that exact adapter for my dev / prototype environment.  It's not as good as the FTDI branded one, but works perfectly fine.

**Comment by tinkerer9** [comment]
Thank you so much!

**Comment by jeffreymonroe** [comment]
I found this [one](https://www.amazon.com/Converter-Adapter-Terminals-Windows-Support/dp/B07SD65BVF/ref=sr_1_22?crid=1FHR9YZJCH185&dib=eyJ2IjoiMSJ9.LQdp_ST1hUETaIor9KnDwvnUYRpfot4xSlRBzaYgjsh3iLOh_UBMcCeo0d79ibHup47VrqOl3JGbeh901PDPgfJVy2PFDTJVFC-NwNFlwQvRzr9aTRr9Vg2U8KCWurm9wlN9oerGkNp8jbppCH8rRpXKbzGo5fX4_tzBOa21S-WqgGbAruaUwWlepJvAhdHdTGr7IeC3qCPRYt_V16m4hy1xtmVewRy_JUUl5ScpG2OqddUIfcgwGv7qVwI4XwCvGj-4jvpZqvmuLzUKzShsjULgKaK_7bgQ4OXWCWglCC8.J2jOoVOGeigHqxI5fpWv6GD1xJPiS1JGgPgh55KDt2U&dib_tag=se&keywords=USB%2Bto%2BRS485%2Badapter&qid=1719240344&s=electronics&sprefix=usb%2Bto%2Brs485%2Badapte%2Celectronics%2C152&sr=1-22&th=1) works well for small form factor applications although it does not have the surge protection that the one referenced above has.

**Comment by tinkerer9** [comment]
Just coming back to confirm that the [adapter](https://www.amazon.com/gp/product/B0BNZBNPWL/ref=ox_sc_act_title_2?smid=A3B0XDFTVR980O&th=1) mentioned above still works over **two years later!**

I just have the `A+` and `B-` connected, with ground disconnected. I have the adapter plug directly into my Raspberry Pi Zero 2 W via the short extension cord included with the adapter.

Thank you so much @sfeakes ❤️!
